src/api/question/dto/update-question.dto.ts

```diff
@@ -1,4 +1,80 @@
-import { PartialType } from '@nestjs/swagger';
-import { CreateQuestionDto } from './create-question.dto';
+import { QuestionDifficulty, QuestionType } from '@common/enums/question.enum'
+import { BooleanField, EnumField, StringField, URLFieldOptional, UUIDField } from '@decorators/field.decorators'
+import { ApiProperty } from '@nestjs/swagger'
+import { Type } from 'class-transformer'
+import { IsArray, IsOptional, ValidateNested } from 'class-validator'
 
-export class UpdateQuestionDto extends PartialType(CreateQuestionDto) {}
+export class UpdateAnswerBodyDto {
+  @ApiProperty({
+    description: 'Answer ID',
+    example: '123e4567-e89b-12d3-a456-426614174000'
+  })
+  @UUIDField()
+  answer_id: string
+
+  @ApiProperty({
+    description: 'Answer content',
+    example: 'Paris'
+  })
+  @StringField()
+  content: string
+
+  @ApiProperty({
+    description: 'Whether this answer is correct',
+    example: true
+  })
+  @BooleanField()
+  isCorrect: boolean
```

Comment:
the `lesson_id` use snake case but why `isCorrect`

```suggestion
  is_correct: boolean
```

because others (class, module, lesson) those have the snake case at the request dto, please help me change all the request dto of the `answer` and `question` into snake case

Reply:
ok, this creates a new scenario — the user can update the corrected answer, but I didn’t handle that case in my code.

Reply:
resolved